examples/campaign_management/validate_text_ad.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example shows use of the validateOnly header for an expanded text ad.
+
+No objects will be created, but exceptions will still be thrown.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, ad_group_id):
+    ad_group_ad_operation = client.get_type('AdGroupAdOperation', version='v2')
+    ad_group_ad = ad_group_ad_operation.create
+    ad_group_service = client.get_service('AdGroupService', version='v2')
+    ad_group_ad.ad_group.value = ad_group_service.ad_group_path(customer_id,
+        ad_group_id)
+    ad_group_ad.status = client.get_type('AdGroupAdStatusEnum',
+        version='v2').PAUSED
+    ad_group_ad.ad.expanded_text_ad.description.value = 'Luxury Cruise to Mars'
```

Comment:
Nit: Put an extra line above this one and add a comment "Create an expanded text ad object." or similar comment.

Reply:
done.